aries_vcx/src/handlers/issuance/issuer.rs

```diff
@@ -164,6 +165,23 @@ impl Issuer {
         .await
     }
 
+    pub async fn send_revocation_notification(&mut self, comment: Option<String>, send_message: SendClosure) -> VcxResult<()> {
+        // TODO: Check if actually revoked
+        if self.issuer_sm.is_revokable() {
+            let rev_msg = RevocationNotification::create()
```

Comment:
I am fine keeping `send_revocation_notification` directly on issuer handler per practicality, but I think it'd be cleaner to have the building of notification message extracted into `protocols/revocation_notification/mod.rs`, similarly like we currently do for ping or oob protocols. 
Then analogically add tests there (as we do for ping) for building both notification and response ack message.